airflow/cli/cli_config.py

```diff
@@ -586,7 +591,6 @@ def string_lower_type(val):
 ARG_CFG_PATH = Arg(("--cfg-path",), help="Path to config file to use instead of airflow.cfg")
 ARG_MAP_INDEX = Arg(("--map-index",), type=int, default=-1, help="Mapped task index")
 
-
```

Comment:
Please revert these whitespace changes. If you are interested in changing them, submit a separate pull request.

Reply:
by mistake it was there, i have deleted that.